src/lib/logger.js

```diff
@@ -1,31 +1,61 @@
+const userLogger = {
+  error: undefined,
+  warn: undefined,
+  debug: undefined
+}
+
 const logger = {
-  error: (errorCode, ...text) => {
-    if (!console) { return }
```

Comment:
I could not understand when console would not be defined, so I took the liberty and removed these checks, also further down in `warn` and `debug`. Please do tell if this has a real importance, and I can add them back in a way that it will not collide with the PR's implementation.